contrib/storage-kafka/src/main/java/org/apache/drill/exec/store/kafka/KafkaPartitionScanSpec.java

```diff
@@ -22,7 +22,7 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class KafkaPartitionScanSpec {
+public class KafkaPartitionScanSpec { //lgtm[java/inconsistent-equals-and-hashcode]
```

Comment:
We should probably fix this.  Inconsistent equals and hashcode could probably result in weird errors.

Reply:
Yeah definitely. I've been a bit split in terms of work so I'll look into this later.